### PR TITLE
Master

### PR DESCRIPTION
**Pull request by kintee**
update versions

### CODE REVIEW

File: template/package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "sinon": "^1.17.6",
     "sinon-chai": "^2.8.0",
     "inject-loader": "*",
-    "isparta-loader": "^2.0.0",
```
Comment (moonou) [inline template/package.json]: update this dependencies will influence `test`, run `npm run unit`  will trigger error

这个工具的变更会影响到测试代码，如果使用`babel-plugin-istanbul`的话会导致`npm run unit`错误

[source code](https://github.com/kintee/webpack/blob/master/template/test/unit/karma.conf.js#L19-L23)

image

File: template/package.json [hunk 2]
```diff
@@ -13,81 +13,82 @@
     "lint": "eslint --ext .js,.vue src{{#unit}} test/unit/specs{{/unit}}{{#e2e}} test/e2e/specs{{/e2e}}"{{/lint}}
   },
   "dependencies": {
-    "vue": "^2.0.1"
```
Comment (moonou) [inline template/package.json]: this dependencies should let user to add

这个依赖应该由用户来添加

**Comment by callumacrae** [comment]
Will this be a breaking change?

**Comment by LinusBorg** [comment]
Sorry, this PR is to outdated. Will close this for now. Also want to clean up dependencies soon.

**Pull request closed without merging**